go.mod

```diff
@@ -37,4 +37,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20220127135604-0fab356f7a15
+
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20220127135755-5e21549c9cb8
```

Comment:
reminder

Reply:
done in https://github.com/codeready-toolchain/registration-service/pull/245/commits/e90410814d45cdc26a617a21769c8bb06b622bd8